lib/zclient.h

```diff
@@ -393,6 +396,7 @@ struct zclient {
 	void (*neighbor_added)(ZAPI_CALLBACK_ARGS);
 	void (*neighbor_removed)(ZAPI_CALLBACK_ARGS);
 	void (*neighbor_get)(ZAPI_CALLBACK_ARGS);
+	int (*gre_update)(ZAPI_CALLBACK_ARGS);
```

Comment:
this returns 'int', unlike most of the others - but that value isn't used, it looks like. if zclient_read can't use the result, maybe it just should be void?

Reply:
ok